### PR TITLE
[feature/OPT-563] to dev

### PR DESCRIPTION
**Pull request by galbiach-irius**

### CODE REVIEW

File: docs/usage/REST-API.md [hunk 1]
```diff
@@ -14,35 +15,211 @@ Options:
   -p, --port INTEGER  StartLeft deployment port.
   --help              Show this message and exit.
 ```
+If not specified, port 5000 will be used by default.
 
-By executing `startleft server` it is possible to see the command-line messages finishing with the following:
+When executing `startleft server` the following command-line message indicates that StartLeft's REST API is ready:
```
Comment (rmenciac) [inline docs/usage/REST-API.md:20]: We should change this sentence. I don't know what it says

Comment (rmenciac) [inline docs/usage/REST-API.md:20]: We should change this sentence. I don't know what it says

File: docs/usage/REST-API.md [hunk 2]
```diff
@@ -14,35 +15,211 @@ Options:
...
     name                        Required. Name of the new project
     mapping_file                Required. File that contains the mapping between IaC resources and threat model resources.
 ```
+This endpoint accepts one or more IaC source files (currently [Cloudformation](../startleft-processors/iac/cft/CloudFormation-Quickstart.md) 
+or [Terraform](../startleft-processors/iac/tf/Terraform-Quickstart.md)) and a mapping file, and generates an OTM with 
+the resulting threat modeling content.
+
+Example request:
+```shell
+curl --location
```
Comment (rmenciac) [inline docs/usage/REST-API.md:75]: There is a "\" missing in all the curl calls

File: docs/usage/REST-API.md [hunk 3]
```diff
@@ -14,35 +15,211 @@ Options:
...
+--request POST localhost:5000/api/v1/startleft/iac \
+--header "Content-Type: multipart/form-data" \
+--header "Accept: application/json" \
+--form iac_type="CLOUDFORMATION" \
+--form iac_file=@"./cft-1.json" \
+--form iac_file=@"./cft-2.json" \
+--form mapping_file=@"./cft-mapping.yaml" \
+--form id="cft-to-otm-example" \
+--form name="CFT to OTM example"
+```
```
Comment (rmenciac) [inline docs/usage/REST-API.md:85]: Maybe we can use real files for the examples, rather than dummy files.

File: docs/usage/REST-API.md (hunk 2 above)
Comment (galbiach-irius) [reply docs/usage/REST-API.md]: Good catch, fixed.

File: docs/usage/REST-API.md (hunk 3 above)
Comment (galbiach-irius) [reply docs/usage/REST-API.md]: Done! I tried to link existing files directly form the repository, but also added the code of some mapping files which are specific to the examples.

**Review by rmenciac — APPROVED** [review]

**Pull request merged**